core/base/skeleton/ZeroSkeleton.cpp

```diff
@@ -531,21 +531,18 @@ int ZeroSkeleton::buildVertexStars(
   printMsg("Building vertex stars", 0, 0, threadNumber_,
            ttk::debug::LineMode::REPLACE);
 
-  vertexStars.resize(vertexNumber);
```

Comment:
where did that go for threadNumber_ != 1?

Reply:
For threadNumber_ > 1, it is done at line 543.
For threadNumber_ == 1, in the std::move at the end of the computation (line 594).

Reply:
gotcha